sovrin_client/test/agent/bulldog_helper.py

```diff
@@ -34,4 +31,4 @@ def getBulldogLogger():
         raise Exception
 
 
-bulldogLogger = getBulldogLogger()
+bulldogLogger = getBulldogLogger
```

Comment:
I think, no one should use this line anymore bulldogLogger = getBulldogLogger, because, whosoever is using that line, meaning, they would be still not using proper basedir path.